src/DataWrangling/ecco_restoring.jl

```diff
@@ -87,7 +87,7 @@ function ecco_times(metadata; start_time = first(metadata).dates)
         push!(times, time)
     end
 
-    return times
+    return tuple(times...)
```

Comment:
the docstring states that this returns an array, but this is a tuple.

Note that using tuples could be problematic if the tuple is really big

Reply:
yep, the slowdown was caused by the tupled times. That should be fixed with https://github.com/CliMA/Oceananigans.jl/pull/3639 so now we can use `CuArray` times